main.py

```diff
@@ -1,10 +1,50 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    try:
+        with open("players.json", "r") as file:
+            data = json.load(file)
```

Comment:
You are overloading the context manager by performing operations inside the 'with' block. According to the checklist, you should exit the block after reading the data. Move the 'data = json.load(file)' line outside of the 'with' block.